src/OrchardCore.Modules/OrchardCore.ContentLocalization/CulturePickerOptions.cs

```diff
@@ -0,0 +1,9 @@
+namespace OrchardCore.ContentLocalization
+{
+    public class CulturePickerOptions
+    {
+        public static readonly int DefaultCookieLifeTime = 14;
+
+        public int CookieLifeTime { get; set; } = DefaultCookieLifeTime;
```

Comment:
```suggestion
        public int CookieLifeTime { get; set; } = 14;
```

Reply:
What's the issue with the `DefaultCookieLifeTime`?!!